src/app/AppController.js

```diff
@@ -0,0 +1,21 @@
+export default class AppController {
+  constructor(model) {
+    this.model = model;
+    this.showNews = this.showNews.bind(this);
+  }
+
+  showNews(event) {
+    const eventTarget = event.target;
+
+    if(eventTarget.matches('.dropdown-content__item')) {
+      const sourceId = eventTarget.getAttribute('id');
+
+      this.model.loadNews(sourceId)
+    }
+  }
+
+  run() {
+    document.querySelector('.nav-wrapper').addEventListener('click', this.showNews);
```

Comment:
per MVC, the View should trigger Controller on user interaction.
In your case, Controller depends on the View interface